src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
```

Comment:
use POST method instead of GET

Reply:
do we usually have to use POST method? in this task, I used GET because we didn't send any secret information. Or user search requests also secret info?